service/assessment/assessment_test.go

```diff
@@ -1038,8 +1039,11 @@ func TestHandleEvidence(t *testing.T) {
 				resourceId: "my-resource-id",
 			},
 			wantErr: func(t assert.TestingT, err error, i ...interface{}) bool {
-				assert.Contains(t, err.Error(), "could not get stream to evidence store")
-				return true
+				if !assert.NotEmpty(t, err) {
```

Comment:
I would check for the specific error message, otherwise we can not be sure if it really fails at that point.